README.md

```diff
@@ -89,11 +89,11 @@ dylib lib("./dynamic_lib.so");
 
 // Get the function adder
 
-auto adder = lib.get_function<double(double, double)>("adder");
+std::function<double(double, double)> adder = lib.get_function<double(double, double)>("adder");
```

Comment:
I'd mention that one can use `auto` for simplicitly - perhaps in a comment or outside of the code snippet.

Alternatively, you could leave this as `auto` and have a comment mention what the actual type is.

Reply:
Done in https://github.com/martin-olivier/dylib/commit/700ff5a75fd4220f80c3521551487c458df9bec1